packages/cli/src/utils/stdout.ts

```diff
@@ -2,9 +2,9 @@ const state: { silent: boolean } = {
   silent: false
 };
 
-export default function log(): void {
+export default function log(...args: any[]): void {
   if (!state.silent && process.env.NODE_ENV !== 'test') {
-    console.log(...arguments);
```

Comment:
Nice! `arguments` always seems too mysterious to me.

Reply:
Yeap! Typescript won't let me call `stdout` with params when the function definition doesn't have parameters defined :slightly_smiling_face: